.github/workflows/tests.yaml

```diff
@@ -44,8 +44,12 @@ jobs:
           python-version: "3.9"
       # Install pylint and kubemarine with all dependencies except ansible.
       # Then uninstall only kubemarine to avoid ambiguity and to surely run pylint on sources.
-      - run: pip install pylint . && pip uninstall -y kubemarine
-      - run: pylint kubemarine scripts test --disable fixme || exit 0
+      - run: pip install .[pylint] && pip install -r requirements-pyinstaller.txt && pip uninstall -y kubemarine
+      - run: |
+          if ! pylint kubemarine scripts ; then
```

Comment:
Are you sure, that we should not check `test` package? Although this is a dev code, it's also read/rewritten/reviewed by developers, for this reason I think, that using linters is a good practice for tests too

Reply:
It introduces ~300 more messages that should be fixed now, and maintained later. Ok, I'll fix them, but what about mypy in this case, do you wish to run it on tests too?